prime-dotnet-webapi/Program.cs

```diff
@@ -39,7 +40,15 @@ public static int Main(string[] args)
 
         public static IHostBuilder CreateHostBuilder(string[] args) =>
             Host.CreateDefaultBuilder(args)
-                .ConfigureWebHostDefaults(webBuilder => webBuilder.UseStartup<Startup>())
+                .ConfigureWebHostDefaults(webBuilder =>
+                {
+                    // webBuilder.ConfigureKestrel(o =>
+                    //     {
+                    //         o.ConfigureHttpsDefaults(o =>
+                    //             o.ClientCertificateMode = ClientCertificateMode.RequireCertificate);
+                    //     });

```

Comment:
Is this commented code intentional?

Reply:
Yes, the webapi should **not** be expecting the client certificate when establishing the HTTPS connection since there will be proxies in between; see https://docs.microsoft.com/en-us/aspnet/core/security/authentication/certauth?view=aspnetcore-5.0#proxy-and-load-balancer-scenarios.  The client certificate will come over a HTTPS header.

But no, we don't need the commented-out code to stay in this file.